pkg/manifests/configfile.go

```diff
@@ -23,15 +24,79 @@ const (
 	CommandTimeout = time.Minute
 )
 
+// This is easier to read as YAML, trust me.
+const configSchemaYAML = `
+"$schema": http://json-schema.org/draft-07/schema#
+definitions:
+  command:
+    type: object
+    required: ['command']
+  version: { const: 1 }
```

Comment:
Shouldn't we bump the version?

Reply:
Why? No-one has used this yet.

Reply:
But we are changing the file format

Reply:
Versioning here is to prevent misunderstandings about what the file means; in particular, to make sure that you aren't giving fluxd an instruction it doesn't know how to carry out.

Can you think of a potential mistake that would be avoided by bumping the version here?

Reply:
I am not absolutely against it, but it does mean writing code that would discriminate between the two versions (assuming we'd want to not invalidate everyone's config immediately.)

Reply:
> Can you think of a potential mistake that would be avoided by bumping the version here?

No, not really.

> I am not absolutely against it, but it does mean writing code that would discriminate between the two versions (assuming we'd want to not invalidate everyone's config immediately.)

Yeah, that's a pain.

I think that, technically, we should bump the version number but I am not sure its worth the pain.

Maybe we could use minor version numbers for backward-compatible syntax changes (without needing to handle it differently in the code). Or maybe we can just leave it at version 1 .